doc/content/getting-started/se-hands-on-guide/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "SENSOR EDITION HANDS ON TUTORIALS"
+description: ""
+weight: 3
+distributions: null
+---
+
+A guide for {{% gnse %}} hands-on videos and tutorials.
+
+<!--more-->
+
+## Introduction to {{% gnse %}}
+
+The video below is the official announcement of {{% gnse %}} at The Things Conference.
```

Comment:
The video below shows the official announcement of {{% gnse %}} at The Things Conference 2021.

Reply:
Done